test/es-module/test-typescript.mjs

```diff
@@ -287,13 +266,12 @@ test('execute a TypeScript test mocking module', { skip: isWindows && process.ar
   strictEqual(result.code, 0);
 });
 
-test('expect process.features.typescript to be false without type-stripping', async () => {
-  strictEqual(process.features.typescript, false);
+test('expect process.features.typescript to be true', async () => {
+  strictEqual(process.features.typescript, 'strip');
```

Comment:
the test title itself here and assertion do not match. `strip` is correct per the docs in the PR
```suggestion
test('expect process.features.typescript to be strip', async () => {
  strictEqual(process.features.typescript, 'strip');
```